lib/plugins/aws/package/compile/events/websockets/lib/stage.js

```diff
@@ -6,6 +6,14 @@ const ServerlessError = require('../../../../../../../serverless-error');
 
 const defaultLogLevel = 'INFO';
 const validLogLevels = new Set(['INFO', 'ERROR']);
+const defaultLogFormat = [
```

Comment:
Hello @Ariette - how did you come up with these defaults? Could you share the source if there's one?

Reply:
The value is come from [the original code](https://github.com/serverless/serverless/blob/main/lib/plugins/aws/package/compile/events/websockets/lib/stage.js#L61)

```javascript
Object.assign(stageResource.Properties, {
        AccessLogSettings: {
          DestinationArn: {
            'Fn::Sub': `arn:\${AWS::Partition}:logs:\${AWS::Region}:\${AWS::AccountId}:log-group:\${${logGroupLogicalId}}`,
          },
          Format: [
            '$context.identity.sourceIp',
            '$context.identity.caller',
            '$context.identity.user',
            '[$context.requestTime]',
            '"$context.eventType $context.routeKey $context.connectionId"',
            '$context.requestId',
          ].join(' '),
        },
        DefaultRouteSettings: {
          DataTraceEnabled: true,
          LoggingLevel: level,
        },
      });
```

Reply:
thanks for clarification, I've missed that 🤦